example-apps/workplace-search/frontend/tailwind.config.js

```diff
@@ -1,55 +1,53 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [
-    "./src/**/*.{js,jsx,ts,tsx}",
-  ],
-  theme: {
-    colors: {
-      // Blues
-      'midnight': '#20377D',
-      'dark-blue': '#005A9E',
-      'blue': '#0077CC',
-      'light-blue': '#1BA9F5',
-      'sky-blue': '#36B9FF',
-      'ice': '#8CD7FF',
-      'blue-tint': '#E8F6FE',
-      // Teals
-      'dark-teal': '#019B8F',
-      'teal': '#00BFB3',
-      'light-teal': '#7DE2D1',
-      'teal-tint': '#E6FAF7',
-      // Poppies
-      'dark-poppy': '#E55940',
-      'poppy': '#FA744E',
-      'light-poppy': '#FF957D',
-      'poppy-tint': '#FDEAE6',
-      // Pinks
-      'dark-pink': '#DD0A73',
-      'pink': '#F04E98',
-      'light-pink': '#F990C6',
-      'pink-tint': '#FCE5F1',
-      // Yellows
-      'dark-yellow': '#F9B110',
-      'yellow': '#FEC514',
-      'light-yellow': '#FFD836',
-      'yellow-tint': '#FFF7D1',
-      // Grays
-      'black': '#000000',
-      'dark-ink': '#1C1E23',
-      'ink': '#343741',
-      'light-ink': '#535966',
-      'lighter-ink': '#7F8494',
-      'dark-smoke': '#949BAB',
-      'smoke': '#ABB4C4',
-      'light-smoke': '#C5CDD8',
-      'dark-fog': '#DCE2EA',
-      'fog': '#E6EBF2',
-      'light-fog': '#F5F7FA',
-      'mist': '#F9FBFC',
-      'white': '#FFFFFF',
-    },
-    extend: {},
-  },
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  // theme: {
```

Comment:
shall we remove this?

Reply:
Yeah, I removed this one on purpuso to use default colors, otherwise there were issue with default colors

Reply:
Got you point. Removed comment as well